src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -2917,7 +2917,15 @@ def test_api_video_update_detail_token_user_upload_state(self):
                 "recording_time": 0,
                 "shared_live_medias": [],
                 "live_state": HARVESTED,
-                "live_info": {},
+                "live_info": {
```

Comment:
In this test, in the `VideoFactory` can you add `started_at` and `stopped_at` in the `live_info` property please ?